api/DocumentFragment.json

```diff
@@ -77,7 +77,7 @@
               "version_added": true
             },
             "safari": {
-              "version_added": false
+              "version_added": true
```

Comment:
I tracked down the WebKit commit and dated it to Safari 8: https://trac.webkit.org/changeset/162062/webkit

```suggestion
              "version_added": "8"
```

Reply:
Thank you!